$OpenBSD: patch-go_client_cmd_update_go,v 1.3 2017/10/11 14:00:05 abieber Exp $

Disabling this because we don't want the binary to update itself
out from under us.

Upstream doesn't seem interested in our hacky Path finding process:
https://github.com/keybase/go-ps/pull/7#discussion_r73917329

Index: go/client/cmd_update.go
--- go/client/cmd_update.go.orig
+++ go/client/cmd_update.go
@@ -7,7 +7,6 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
-	"os/exec"

 	"github.com/keybase/cli"
 	"github.com/keybase/client/go/install"
@@ -35,23 +34,7 @@ func newCmdUpdateCheck(cl *libcmdline.CommandLine, g *
 		Name:  "check",
 		Usage: "Check for update",
 		Action: func(c *cli.Context) {
-			if libkb.IsBrewBuild {
-				g.Log.Errorf("\nTo update, run:\n\n\tbrew upgrade keybase")
-				return
-			}
-
-			updaterPath, err := install.UpdaterBinPath()
-			if err != nil {
-				g.Log.Errorf("Error finding updater path: %s", err)
-				return
-			}
-
-			cmd := exec.Command(updaterPath, "check")
-			cmd.Stdout = os.Stdout
-			cmd.Stderr = os.Stderr
-			if err := cmd.Run(); err != nil {
-				g.Log.Errorf("Error running %q: %s", updaterPath, err)
-			}
+			g.Log.Errorf("Not supported on OpenBSD.")
 		},
 	}
 }
